### PR TITLE
Annie/ner versioning

### PR DESCRIPTION
**Pull request by anneCarlson**
Reviewers: 
----------

- [x] @madisonmay 
- [ ] @anneCarlson

What's the purpose of the PR?
-----------------------------

Updates version number of ner.
https://trello.com/c/voWeqH80/1986-3-ner-updated-in-client-libs-default-version
Fix for custom domain is in there as well.

Description of changes
----------------------

Notes to reviewers
------------------

Keep in mind while reviewing code:
----------------------------------
- Is relevant code tested?
- Are added functions/methods documented?
- Separation of concerns (SOC)
- Don't repeat yourself (DRY) 
- Limit the number of positional arguments
- Is function/method length reasonable?
- Can code be broken down into smaller components?
- Where do added functions/methods belong?
- Are variable names descriptive?
- Are errors handled appropriately?
- Can logic be simplified?
- Does the code make sense in context? (expand the diff)

### CODE REVIEW

**Comment by sihrc** [comment]
This needs to be rebased against development. There's a bunch of extra PR commits.

File: lib/indico.rb [hunk 1]
```diff
@@ -102,15 +102,24 @@ def self.text_features(text, config = nil)
     return api_handler(text, 'textfeatures', config)
   end
 
-  def self.people(text, config = nil)
+  def self.people(text, config = {})
+    unless config.key?('v') or config.key?('version')
```
Comment (sihrc) [inline lib/indico.rb]: we're no longer using unless. we use if.

**Comment by anneCarlson** [comment]
Yeah, I think I accidentally branched off master. I'll do that.

File: lib/indico.rb (hunk 1 above)
Comment (anneCarlson) [inline lib/indico.rb]: Just as a hard rule? That makes this line so much more convoluted...

**Comment by madisonmay** [comment]
LGTM

**Pull request merged**